test/index.js

```diff
@@ -91,7 +91,7 @@ describe('Likely', function () {
         it('should change the shared URL when `<link rel="canonical">` is specified', function () {
             return driver.executeScript(`
                 document.head.innerHTML += '<link rel="canonical" href="https://google.com">';
-                likely.initiate();
+                likely.default.initiate();
```

Comment:
Okay. But I think this is temporary solution. It will strike again when we will move to webpack@2.